pycbc/events/coinc.py

```diff
@@ -701,11 +709,62 @@ def __init__(self, num_templates, analysis_block, background_statistic,
 
         self.singles = {}
 
-        #if save_background_on_interrupt:
-        #    import signal
-        #    def sig_handler(signum, frame):
-        #        pass
-        #    signal.signal(signal.SIGINT, sig_handler)
+    # Adding this here as it requires the results format dict as defined by this
+    # class.
+    @classmethod
+    def pick_best_coinc(cls, coinc_results):
+        """Choose the best two-ifo coinc by ifar first, then statistic if needed.
+
+        This function picks which of the available double-ifo coincs to use.
+        It chooses the best (highest) ifar. The ranking statistic is used as
+        a tie-breaker.
+        A trials factor is applied if multiple types of coincs are possible
+        at this time given the active ifos.
+
+        Parameters
+        ----------
+        coinc_results: list of coinc result dicts
+            Dictionary by detector pair of coinc result dicts.
+
+        Returns
+        -------
+        best: coinc results dict
+            If there is a coinc, this will contain the 'best' one. Otherwise
+            it will return the provided dict.
+        """
+        mstat = 0
+        mifar = 0
+        mresult = None
+
+        # record the trials factor from the possible coincs we could
+        # maximize over
+        trials = 0
+        for result in coinc_results:
+            # Check that a coinc was possible. This was added in the
+            # 'add_singles' method of this class
```

Comment:
'this was added' is irrelevant - to refer people to another bit of code just say "see the add_singles method"